From 200a5feb87674563b8c5bca8d583a0fcbb981539 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 16 Jul 2018 13:52:11 +0800
Subject: [PATCH 4262/5242] MLK-18605-9 drm/bridge: sec-dsim: increase timeout
 values for BTA and LPRX

commit  31731dc2da232da35ba8218b0925f1535620465e from
https://source.codeaurora.org/external/imx/linux-imx.git

The current timeout values for 'BTA' and 'LPRX' in register
'DSIM_TIMEOUT' is not long enough for some dsi peripherals,
so increase them long enough for all current peripherals to
avoid timeout errors generation in some cases.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 8e2b86b317382635e74002a5b2a466804247e61a)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/sec-dsim.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index e344dc8..56d17dc 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -609,9 +609,9 @@ static void sec_mipi_dsim_config_dphy(struct sec_mipi_dsim *dsim)
 		      PHYTIMING2_SET_M_THSTRAILCTL(11);
 	dsim_write(dsim, phytiming2, DSIM_PHYTIMING2);
 
-	timeout |= TIMEOUT_SET_BTAOUT(0xf)	|
-		   TIMEOUT_SET_LPDRTOUT(0xf);
-	dsim_write(dsim, 0xf000f, DSIM_TIMEOUT);
+	timeout |= TIMEOUT_SET_BTAOUT(0xff)	|
+		   TIMEOUT_SET_LPDRTOUT(0xff);
+	dsim_write(dsim, timeout, DSIM_TIMEOUT);
 }
 
 static void sec_mipi_dsim_init_fifo_pointers(struct sec_mipi_dsim *dsim)
-- 
1.7.9.5

